sklearn/linear_model/_least_angle.py

```diff
@@ -22,9 +22,20 @@
 from ..model_selection import check_cv
 
 # mypy error: Module 'sklearn.utils' has no attribute 'arrayfuncs'
-from ..utils import arrayfuncs, as_float_array, check_random_state  # type: ignore
+from ..utils import (  # type: ignore
```

Comment:
O_o why do we need the type ignore here and not in other places

Reply:
I think this was already present.

Reply:
It is written above:

```
# mypy error: Module 'sklearn.utils' has no attribute 'arrayfuncs'
```